dev/MRTCore/packaging/MrtCore.PriGen.targets

```diff
@@ -11,8 +11,8 @@
     <!-- Any .NET 5 or greater version is valid here! -->
     <TargetPlatformIdentifierAdjusted Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' and $([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), '5.0'))">UAP</TargetPlatformIdentifierAdjusted>
 
-    <!-- Add support for console apps. -->
-    <TargetPlatformIdentifierAdjusted Condition="'$(OutputType)' == 'Exe'">UAP</TargetPlatformIdentifierAdjusted>
+    <!-- Add support for console apps and libraries. -->
+    <TargetPlatformIdentifierAdjusted Condition="'$(OutputType)' == 'Exe' or '$(OutputType)' == 'Library'">UAP</TargetPlatformIdentifierAdjusted>
```

Comment:
>TargetPlatformIdentifierAdjusted [](http://example.com/codeflow?start=5&length=32)

if this is not UAP, what's the result of this file?

Reply:
Build error: error APPX3217: SDK folder containing 'Windows.props' for 'Windows 10.0.19041.0' cannot be located.

Reply:
then why do we need to the condition?

Reply:
I'm sorry, I don't understand. We need to change the TPI to UAP to avoid this error. There may be other places in the managed helper code too where there is a dependency on the TPI being UAP.

Reply:
if this code only works when TargetPlatformIdentifierAdjusted is UAP, we need to set TargetPlatformIdentifierAdjusted to UAP unconditionally to make this file at least self consistent. If for any TPI this is undesirable, it needs to be filtered to not import this targets file.

Reply:
OK, I unconditionally set it in the file now. The import is already done conditionally. Right now, we don't do it only for WAPPROJ.